modules/cluster-logging-collector-log-forward-sts-cloudwatch.adoc

```diff
@@ -53,59 +55,102 @@ $ ccoctl aws create-iam-roles \
 --name=<name> \
 --region=<aws_region> \
 --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
---identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com <1>
+--identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com #<1>
 ----
 <1> <name> is the name used to tag your cloud resources and should match the name used during your STS cluster install
-+
-. Apply the secret created:
+
+. Apply the secret by running the folloing command:
 [source,terminal]
 +
 ----
 $ oc apply -f output/manifests/openshift-logging-<your_role_name>-credentials.yaml
 ----
+
+. Create a credentials secret:
 +
+[source,yaml]
+----
+apiVersion: v1
+stringData:
+  credentials: |-
+    [default]
+    sts_regional_endpoints = regional
+    role_arn = <MY ROLE ARN>
+    web_identity_token_file = <path_to_sa token>
+kind: Secret
+metadata:
+  name: sts-secret
+  namespace: openshift-logging
+type: Opaque
+----
+
+. Apply the generated credentials secret by running the following command: 
++
+[source,terminal]
+----
+$ oc apply -f openshift-logging-sts-secret-credentials.yaml
+----
+
 . Create or edit a `ClusterLogForwarder` custom resource:
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
```

Comment:
this feature works for any namespace, as before.  Not only openshift-logging.   The credential requests and secret, SA, role, etc, do not have to be tied to the openshift-logging ns.   should we include a note?